src/lib/plugin_apis/fs.api

```diff
@@ -409,6 +409,32 @@ gboolean bd_fs_clean (const gchar *device, GError **error);
  */
 gchar* bd_fs_get_fstype (const gchar *device,  GError **error);
 
+/**
+ * bd_fs_freeze:
+ * @device: the device to freeze (device must be mounted, trying to freeze an
+ *          unmounted device will result in an error)
+ * @error: (out): place to store error (if any)
+ *
+ * Freezes filesystem on @device. The filesystem must support freezing.
+ *
+ * Returns: whether @device was successfully freezed or not
+ *
+ */
+gboolean bd_fs_freeze (const gchar *device, GError **error);
+
+/**
+ * bd_fs_unfreeze:
+ * @device: the device to un-freeze (device must be mounted, trying to un-freeze
+ *          an unmounted device will result in an error)
+ * @error: (out): place to store error (if any)
+ *
+ * Un-freezes filesystem on @device. The filesystem must support un-freezing.
```

Comment:
I think it would make more sense to say that the FS needs to support freezing here. :smile:

Reply:
Fixed.